server/router.js

```diff
@@ -1,17 +1,28 @@
 const express = require('express');
-const db = require('../db/database.js');
+const db = require('../db/queries.js');
 
 const router = express.Router();
 
-router.get('/:id', (req, res) => {
-  res.set({ 'Access-Control-Allow-Origin' : '*' });
-  db.find(req.params, (err, data) => {
+router.get('/:id', async (req, res) => {
+  const { id } = req.params;
+  db.findOnePostGres(id, (err, data) => {
     if (err) {
       res.sendStatus(404);
     } else {
-      res.send(data);
+      res.send(data[0]);
     }
-  })
+  });
 });
 
+// router.get('/:id', (req, res) => {
```

Comment:
I would go ahead and remove this dead code unless it's going to be developed futher